lib/Elastica/Type.php

```diff
@@ -299,6 +299,22 @@ public function setMapping($mapping)
     }
 
     /**
+     * Sets value type mapping for this type with timestamp .
+     *
+     * @param \Elastica\Type\Mapping|array $mapping Elastica\Type\MappingType object or property array with all mappings
+     *
+     * @return \Elastica\Response
+     */
+    public function setMappingWithTimeStamp($mapping,$timestamp)
```

Comment:
I'm not sure is this method needed? Mapping know about timestamp, Type know only about mapping, but not mapping details.